source/read-write-concern/read-write-concern.rst

```diff
@@ -131,6 +136,14 @@ considered the server’s default ``ReadConcern``.
 default ``ReadConcern`` while the latter is the user explicitly specifying a
 ``ReadConcern`` with a ``level`` of “local”.
 
+Snapshot Read Concern
+---------------------
+
+When a ``ReadConcern`` ``level`` ``snapshot`` is used, ``atClusterTime`` may be specified to indicate
+the desired point in time for reading. ``find`` and ``aggregate`` operations executed with ``ReadConcern`` ``snapshot`` but without ``atClusterTime``
```

Comment:
I assume `distinct` is not mentioned here because it does not have a `cursor` field in the reply and `atClusterTime` would just be in the top level?